src/main/java/school/lemon/changerequest/java/introduction/hw2/Task8.java

```diff
@@ -3,6 +3,24 @@
 public class Task8 {
 
     public static int[] evenOdd(int[] arr) {
-        return null;
+        if (arr == null) {
+            return null;
+        }
+        int lengthArr = arr.length;
+        int result[] = new int[lengthArr];
+        int j = 0;
+        for (int i = 0; i < lengthArr; i++) {
+            if (arr[i] % 2 == 0) {
+                result[j++] = arr[i];
+            }
+        }
+        int k = 0;
+        for (int i = 0; i < lengthArr; i++) {
+            if (arr[i] % 2 != 0) {
```

Comment:
It's quite not right way to check values, you should always try to solve task without negation, as negation is quite harder to understand.
Here code will be clearer with `arr[i]%2==1`